jenkins_pipelines/manager_prs/Jenkinsfile_prs_check

```diff
@@ -9,7 +9,7 @@ properties([
 pipeline {
 
     parameters {
-        string(defaultValue: "", description: 'Gitarro PR', name: 'GITARRO_PR_NUMBER')
+        string(defaultValue: '167', description: 'Gitarro PR', name: 'GITARRO_PR_NUMBER')
```

Comment:
Why do we want `167` as default value?

Reply:
We don't. That's what I told you we need to keep in mind and remove just after merge gitarro, so meanwhile is Jenkins CI is working fine using that fork and that gitarro changes